api/app/Repositories/Event/EventRepository.php

```diff
@@ -63,4 +63,14 @@ public function getEventsEmails(
             ->distinct()
             ->get();
     }
+
+    public function getById(int $id): ?Event
+    {
+        return Event::find($id);
+    }
+
+    public function deleteById(int $id): void
+    {
+        Event::destroy($id);
```

Comment:
I thinnk, we should cancel events in application, not delete

Reply:
Change to cancelled status + added reschedule ability